transports/quic/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "libp2p-quic"
-version = "0.7.0-alpha"
+version = "0.7.1-alpha"
```

I think the appropriate bump would be:


```suggestion
version = "0.7.0-alpha.2"
```

`0.7.1-alpha` suggests that a `0.7.0` exists and we are pre-released a patch version (although the non-breaking aspect of said patch version is not guaranteed, hence the alpha).

We haven't even released `0.7.0` so we may as well push more alpha's out until we are happy with it.

---

Ah yes you are right! Fixed in 2a83d3c. This also means that we don't need to bump the version in the `libp2p` crate.

---

> Ah yes you are right! Fixed in 2a83d3c. This also means that we don't need to bump the version in the `libp2p` crate.

I think you will have to, I don't think alpha versions update automatically. AFAIK, they are basically major version bumps but with an "unstable" co-notation.

---

I could be wrong though, please test it :)